test/scripts/scripts.test.js

```diff
@@ -31,10 +38,11 @@ describe('Decorating', () => {
   });
 
   it('Loads lana.js upon calling lana.log the first time', async () => {
+    await delay(200); // wait for non-blocking imports in loadArea()
```

Comment:
Can we prevent the delay by having an intersection observer, or waiting until window.lana.log actually exists?
This might still pop up as failing on a slow CI day otherwise